src/core/models/verticalresults.js

```diff
@@ -42,21 +42,22 @@ export default class VerticalResults {
    * @param {VerticalResults} verticalResults
    * @param {Object<string, string>} urls keyed by vertical key
    * @param {ResultsContext} resultsContext
+   * @param {String} verticalKey
    * @returns {@link Section}
    */
-  static fromCore (verticalResults, urls = {}, resultsContext = ResultsContext.NORMAL) {
+  static fromCore (verticalResults, urls = {}, resultsContext = ResultsContext.NORMAL, verticalKey) {
     if (!verticalResults) {
       return new Section();
     }
 
     return new Section(
       {
-        verticalConfigId: verticalResults.verticalKey,
+        verticalConfigId: verticalResults.verticalKey || verticalKey,
```

Comment:
Is `verticalResults.verticalKey` sometimes not present?

Reply:
Right! It looks like it's not present in vertical search responses (though we supply it in the request), but it is present in `alternativeVerticals` and universal result's `verticalResults`. This is something that existed in the non-answers-core branches of the SDK, it was removed in this PR https://github.com/yext/answers/pull/1193. I can move the change over to that PR so it doesn't seem facets-specific.